crates/core/src/client/constants.rs

```diff
@@ -1,13 +1,12 @@
 pub const CHAIN_ID: u64 = 1_263_227_476;
 
-pub const STARKNET_NATIVE_TOKEN: &str =
-    "0x49d36570d4e46f48e99674bd3fcc84644ddd6b96f7c741b1562b82f9e004dc7";
+pub const STARKNET_NATIVE_TOKEN: &str = "0x49d36570d4e46f48e99674bd3fcc84644ddd6b96f7c741b1562b82f9e004dc7";
 
 pub mod selectors {
-    use starknet::{core::types::FieldElement, macros::selector};
+    use starknet::core::types::FieldElement;
```

Comment:
interesting that the way that doesn't share structure is preferred!

Reply:
```toml
edition = "2021"
newline_style = "Unix"
use_field_init_shorthand = true
use_small_heuristics = "Max"
use_try_shorthand = true
max_width = 120

# Unstable features below
unstable_features = true
version = "Two"
comment_width = 100
format_code_in_doc_comments = true
format_macro_bodies = true
format_macro_matchers = true
format_strings = true
imports_granularity = "Module"
group_imports = "StdExternalCrate"
normalize_comments = true
normalize_doc_attributes = true
wrap_comments = true

```